src/redux/reducers/loginReducer.js

```diff
@@ -8,9 +8,7 @@ import {
   SOCIAL_AUTH_SUBMIT,
   SOCIAL_AUTH_SUCCESS,
 } from '../actions-types';
-import { login as initialState } from '../initialState.json';
-
-// const { login } = initialState;
```

Comment:
Remove this comment.

Reply:
@NkFab That comment came from the develop branch. It is highlighted in red because I corrected it

Reply:
My bad I missed that part.